_docs/concepts/security/rbac.md

```diff
@@ -0,0 +1,221 @@
+   kind: ServiceRole
+   metadata:
+     name: products-viewer
+     namespace: default
+   spec:
+     rules:
+     - services: ["products.svc.cluster.local"]
+       methods: ["GET", "HEAD"]
+```
+
+In addition, we support **prefix match** and **suffix match** for all the fields in a rule. For example, you can define a "tester" role that
+has the following permissions in "default" namespace:
+* Full access to all services with prefix "test-" (e.g, "test-bookstore", "test-performance", "test-api.svc.cluster.local").
+* Read ("GET") access to all paths with "/reviews" suffix (e.g, "/books/reviews", "/events/booksale/reviews", "/reviews") in service "bookstore.svc.cluster.local".
+
+```rule
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: ServiceRole
+   metadata:
+     name: tester
+     namespace: default
+   spec:
+     rules:
+     - services: ["test-*"]
+       methods: ["*"]
+     - services: ["bookstore.svc.cluster.local"]
+       paths: ["*/reviews"]
+       methods: ["GET"]
+```
+
+In ServiceRole, the combination of "namespace"+"services"+"paths"+"methods" defines "how a service (services) is allowed to be accessed".
+In some situations, you may need to specify additional constraints that a rule applies to. For example, a rule may only applies to a
+certain "version" of a service, or only applies to services that are labeled "foo". You can easily specify these constraints using
+custom fields.
+
+For example, the following ServiceRole definition extends the previous "products-viewer" role by adding a constraint on service "version"
+being "v1" or "v2". Note that the "version" property is provided by "action.properties.version" in "requestcontext".
+
+```rule
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: ServiceRole
+   metadata:
+     name: products-viewer-version
+     namespace: default
+   spec:
+     rules:
+     - services: ["products.svc.cluster.local"]
+       methods: ["GET", "HEAD"]
+       constraints:
+       - key: "version"
+         values: ["v1", "v2"]
+```
+
+### ServiceRoleBinding
+
+A ServiceRoleBinding specification includes two parts:
+* `roleRef` refers to a ServiceRole object **in the same namespace**.
+* A list of `subjects` that are assigned the role.
+
+A subject can either be a "user", or a "group", or is represented with a set of "properties". Each entry ("user" or "group" or an entry
+in "properties") must match one of fields ("user" or "groups" or an entry in "properties") in the "subject" part of the "requestcontext"
+instance.
+
+Here is an example of ServiceRoleBinding object "test-binding-products", which binds two subjects to ServiceRole "product-viewer":
+* user "alice@yahoo.com".
+* "reviews" service in "abc" namespace.
+
+```rule
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: ServiceRoleBinding
+   metadata:
+     name: test-binding-products
+     namespace: default
+   spec:
+     subjects:
+     - user: "alice@yahoo.com"
+     - properties:
+         service: "reviews"
```

Comment:
In the ServiceRole examples you used fully qualified domain names for services, but here just a short name. Which one is correct? You should update the incorrect example.

Reply:
Updated to "reviews.abc.svc.cluster.local".